server/lib/routes/forests.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const Router = express.Router;
+const router = Router();
+const client = require('../../db-client');
+
+router
+  .get('/', (req, res) => {
+    client.query(`
+      SELECT id, name, forest, sites, rvwaste
+      FROM campgrounds;
+    `)
+      .then(result => {
+        res.json(result.rows[0]);
```

Comment:
```suggestion
        res.json(result.rows);
```
The code before was only returning one row of campgrounds, whereas the code above will return all of them.